src/raftstore/store/fsm/store.rs

```diff
@@ -430,7 +438,7 @@ impl<'a, T: Transport, C: PdClient> StoreFsmDelegate<'a, T, C> {
         RAFT_EVENT_DURATION
             .with_label_values(&[tick.tag()])
             .observe(duration_to_sec(t.elapsed()) as f64);
-        slow_log!(t, "{} handle timeout {:?}", self.fsm.store.tag, tick);
+        slow_log!(t, "[store {}] handle timeout {:?}", self.fsm.store.id, tick);
```

Comment:
I don't think this is gonna to work (correctly parsed as a field). Please modify the `slow_log` implementation instead.

Reply:
I plan to refactor `slow_log` like this:
```rust
macro_rules! slow_log {
    ($t:expr, $($arg:tt)*) => {{
        if $t.is_slow() {
            warn!("handle task slow";
                "task" => format!("{}", format_args!($($arg)*)),
                "elapsed" => ?$t.elapsed(),
            );
        }
    }}
}
```
Is it ok?

Reply:
Maybe we should use the `takes` instead of the `elapsed`, for backward compatibility.

Reply:
How about "handle task takes long" 🤣

Actually, the keyword for slow is really "slow" now ("handle task slow" in slow_log and "slow_query" in coprocessor)?